src/Core/Grid/Definition/Factory/CurrencyGridDefinitionFactory.php

```diff
@@ -74,10 +79,23 @@ protected function getName()
     protected function getColumns()
     {
         return (new ColumnCollection())
-            ->add((new DataColumn('currency'))
+            ->add(
+                (new BulkActionColumn('currency_bulk'))
+                    ->setOptions([
+                        'bulk_field' => 'id_currency',
+                    ])
+            )
+            ->add(
+                (new DataColumn('id_currency'))
+                    ->setName($this->trans('ID', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'id_currency',
+                    ])
+            )
+            ->add((new DataColumn('name'))
```

Comment:
Be careful since you changed the column here, this will remove a behaviour from https://github.com/PrestaShop/PrestaShop/blob/3112ecde59959ab42fa6ee9ea23fcdd9278258c2/src/Core/Currency/CurrencyGridDataFactory.php#L93

It is supposed to add `Edited` when a currency has been edited (different data than CLDR repository), and it adds an icon for unofficial currencies. It's not the cleanest implementation I admit (especially the hard coded HTML for icon), a better way to do this would be to create a custom column twig but I struggled with it I don't remember why.. 🤔

Anyway you can just fix this by changing the `CurrencyGridDataFactory` and update the `name` field instead of `currency` in the records data.

Reply:
Nice catch @jolelievre  ! Thank you so much man.